libbeat/outputs/mode/mode_test.go

```diff
@@ -1,309 +0,0 @@
-// +build !integration
```

Comment:
This file did not contain any tests or any piece of code used somewhere. Missing delete from the past.